README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion - **The use of a method/application inside itself**
+- Recursive Case - **A problem that is divided into smaller problems**
+- Base Case - **An immediately solvable problem**
+- Activation Chain/Stack - **All of the iterations of the method being called within itself**
+- Activation Record/Call - **A single instance/iteration point of the method being called within itself**
+- Infinite Recursion/Stack Overflow/Stack too deep - **An error that occurs when the method results in an infinite loop**
```

Comment:
I wouldn't say "error". It's a situation where the method results in an excessive use of memory that is usually caused by an infinite loop of sorts with many many calls to the recursive method.